src/components/displaybooks.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import style from './displaybook.module.css';
+
+const DisplayBooks = (props) => (
+  <div className={style.book}>
+    <div className={style.aboutBook}>
+      <p>{props}</p>
+      <h2>{props}</h2>
+      <a href="#head">{props}</a>
+      <div className={style.bookAction}>
+        <a href="#comments">Comments</a>
+        {' '}
+        |
+        <a href="#remove">Remove</a>
+        |
+        <a href="#edit">Edit</a>
+      </div>
+    </div>
+
+    <div className={style.progress}>
+      <progress value={props} max="100" />
+      <div className={style.progressDetail}>
+        <h4>{props}</h4>
+        <span>Completed</span>
+      </div>
+    </div>
+
+    <div className={style.currentChapter}>
+      <p>Current Chapter</p>
+      <h4>{props}</h4>
+      <button type="button">Update Progress</button>
+    </div>
+  </div>
+);
```

Comment:
- [ ] You are receiving props in this component but you are not passing the right values into the fields they are supposed to be. I see you're just passing props in the different elements. But I believe you were supposed to do it like this: for the case of the book title, `props.book.title`, author, and other related fields you call the field name after the `props.book.fieldname`. Or better, you could still destructure the values you want from the props and display them in their respective places. Kindly work on this so the app displays something. 👍🏽


[ Optional Changes ]
- [ ] What if you could rename this file to be a single book? Looking at it, it's actually supposed to be `Book.js` to represent a single book 👍🏽

Reply:
Hello @AmaduKamara 
Given that we are simply just initializing the project, it is not directly required that we pass the right values now. The project simply requires that we initialize the project without any functionalities.

Reply:
When I say the right value, I meant, the value passed into the single book component when you looped over the array. Or better, you could have passed a simple prop value instead of the looked array. 
Currently, the app is not running or displaying anything in the browser because you are not making use of the props values.

Reply:
If the app is not rendering anything on the screen, then the work is incomplete. It's expected to render a navigation bar, and display static dummy books. But at the moment your app is not rendering anything to the screen and it's showing errors in the console.

Reply:
Yeah, I did it that way before but I'd to redo it in this manner because the linters kept failing. The only other was for me to disable the linters for the file but it would've still resulted in some required changes.